### PR TITLE
tests: interfaces tests normalization

### PR DESCRIPTION
**Pull request by sergiocazzolato**
This change sets similat structure for all the interfaces tests.

Also it removes not needed PWD variables

### CODE REVIEW

File: tests/main/interfaces-system-observe/task.yaml [hunk 1]
```diff
@@ -42,26 +42,29 @@ execute: |
         su -l -c "test-snapd-system-observe-consumer.dbus-introspect" test | MATCH "$expected"
     fi
 
-    if [ "$(snap debug confinement)" = strict ] ; then
-        echo "And the policy has the ptrace suppression rule"
-        MATCH '^deny ptrace \(trace\),' < /var/lib/snapd/apparmor/profiles/snap.test-snapd-system-observe-consumer.consumer
+    if [ "$(snap debug confinement)" = partial ] ; then
```
Comment (bboozzoo) [inline tests/main/interfaces-system-observe/task.yaml]: Mentioned this in another review, but we'll probably need to start looking at `snap debug sandbox-features` and making sure that apparmor profile is not downgraded. This way we can check the confinement on systems where AppArmor is supported by snapd.

File: tests/main/interfaces-bluetooth-control/task.yaml [hunk 2]
```diff
@@ -55,3 +32,31 @@ execute: |
     echo "And the snap is able to read dev"
     cat "$BTDEV"/*/power/control | tee control
     [ "$(su -l -c '/snap/bin/generic-consumer.cmd cat '"$BTDEV"'/*/power/control' test)" = "$(cat control)" ]
+
+    echo "When the plug is disconnected"
+    snap disconnect generic-consumer:bluetooth-control
+
+    if [ "$(snap debug confinement)" = partial ] ; then
```
Comment (bboozzoo) [inline tests/main/interfaces-bluetooth-control/task.yaml]: I other tests, the disconnect happens after partial confinement test.

**Comment by bboozzoo** [comment]
Some tests need looking into:
```
2019-01-22 22:09:21 Failed tasks: 30
    - google:amazon-linux-2-64:tests/main/interfaces-system-observe
    - google:arch-linux-64:tests/main/interfaces-system-observe
    - google:centos-7-64:tests/main/interfaces-system-observe
    - google:debian-9-64:tests/main/interfaces-system-observe
    - google:fedora-29-64:tests/main/interfaces-system-observe
    - google:ubuntu-14.04-64:tests/main/interfaces-avahi-observe
    - google:ubuntu-14.04-64:tests/main/interfaces-system-observe
    - google:ubuntu-16.04-32:tests/main/interfaces-avahi-observe
    - google:ubuntu-16.04-32:tests/main/interfaces-fuse_support:parallel
    - google:ubuntu-16.04-32:tests/main/interfaces-fuse_support:regular
    - google:ubuntu-16.04-32:tests/main/interfaces-system-observe
    - google:ubuntu-16.04-64:tests/main/interfaces-avahi-observe
    - google:ubuntu-16.04-64:tests/main/interfaces-fuse_support:parallel
    - google:ubuntu-16.04-64:tests/main/interfaces-fuse_support:regular
    - google:ubuntu-16.04-64:tests/main/interfaces-system-observe
    - google:ubuntu-18.04-64:tests/main/interfaces-avahi-observe
    - google:ubuntu-18.04-64:tests/main/interfaces-fuse_support:parallel
    - google:ubuntu-18.04-64:tests/main/interfaces-fuse_support:regular
    - google:ubuntu-18.04-64:tests/main/interfaces-system-observe
    - google:ubuntu-18.04-64:tests/unit/spread-shellcheck
    - google:ubuntu-18.10-64:tests/main/interfaces-avahi-observe
    - google:ubuntu-18.10-64:tests/main/interfaces-fuse_support:parallel
    - google:ubuntu-18.10-64:tests/main/interfaces-fuse_support:regular
    - google:ubuntu-18.10-64:tests/main/interfaces-system-observe
    - google:ubuntu-core-16-64:tests/main/interfaces-fuse_support:parallel
    - google:ubuntu-core-16-64:tests/main/interfaces-fuse_support:regular
    - google:ubuntu-core-16-64:tests/main/interfaces-system-observe
    - google:ubuntu-core-18-64:tests/main/interfaces-fuse_support:parallel
    - google:ubuntu-core-18-64:tests/main/interfaces-fuse_support:regular
    - google:ubuntu-core-18-64:tests/main/interfaces-system-observe
```

File: tests/main/interfaces-fuse_support/task.yaml [hunk 3]
```diff
@@ -19,7 +19,7 @@ environment:
     MOUNT_POINT_OUTSIDE/regular: /var/snap/test-snapd-fuse-consumer/current/mount_point
     NAME/regular: test-snapd-fuse-consumer
     # snap with instance key 'foo'
-    MOUNT_POINT/parallel: /var/snap/test-snapd-fuse-consumer/current/mount_point
+    MOUNT_POINT/parallel: /var/snap/test-snapd-fuse-consumer_foo/current/mount_point
```
Comment (mvo5) [inline tests/main/interfaces-fuse_support/task.yaml]: Nice catch.

**Pull request merged**